MISC/ShootSingleBulletBetweenCoordsIgnoreEntityNew.md

```diff
@@ -6,32 +6,31 @@ aliases: ["0xBFE5756E7407064A","_SHOOT_SINGLE_BULLET_BETWEEN_COORDS_WITH_EXTRA_P
 
 ```c
 // 0xBFE5756E7407064A 0xCCDC33CC
-void SHOOT_SINGLE_BULLET_BETWEEN_COORDS_IGNORE_ENTITY_NEW(float x1, float y1, float z1, float x2, float y2, float z2, int damage, BOOL p7, Hash weaponHash, Ped ownerPed, BOOL isAudible, BOOL isInvisible, float speed, Entity entity, BOOL p14, BOOL p15, BOOL p16, BOOL p17);
+void SHOOT_SINGLE_BULLET_BETWEEN_COORDS_IGNORE_ENTITY_NEW(float x1, float y1, float z1, float x2, float y2, float z2, int damage, BOOL pureAccuracy, Hash weaponHash, Ped ownerPed, BOOL isAudible, BOOL isInvisible, float speed, Entity entity, BOOL p14, BOOL p15, BOOL p16, BOOL p17, Any p18, Any p19, Any p20);
```

Comment:
Remove these parameters you added `, Any p18, Any p19, Any p20`, but keep `pureAccuracy`.